tests/test_integration.py

```diff
@@ -445,10 +447,10 @@ async def test_info(envsonschema):
     assert proc.returncode == 0, (stdout, stderr)
     assert stdout.decode() == dedent(
         """\
-        name: "envsonschema"
-        language: "python"
-        issues: "https://github.com/bowtie-json-schema/bowtie/issues"
-        dialects: [
+        name:"envsonschema"
```

Comment:
The fact you had to modify this should have pointed out you're now missing a space :) -- use the tests to find bugs! That's what they're there for.